### ISSUE TITLE
[Quest][Dire Maul] Free Knot!, reputation reward

### ISSUE THREAD

**Issue by Nihilia** [FeenixServerProject/Archangel_2.4.3_Bugtracker#3306]
**Issue:**
Upon turning quest [Free Knot!](http://www.wowhead.com/quest=5525) (and its repeatable version [Free Knot!](http://www.wowhead.com/quest=7429)) player receives 350 Reputation point with Steamwheedle Cartel (SC for shorten), non-existing fraction. Gained reputation also does not appear in fraction window, however these virtual points do stack somewhere, and player can receive messages like "You became Friendly with SC," and so on.

**How it should be:**
[Steamwheedle Cartel](http://www.wowwiki.com/Steamwheedle_Cartel) is not a fraction in usual meaning of this word. SC is same thing you would call Alliance or Horde, its a union of fractions. It unites goblins of four main houses: Booty Bay, Everlook, Ratchet and Gadgetzan. If you open your reputation window, all these goblin fractions are lined under SC heading.
In other words: reputation gained for SC, splits in four fractions equally. For example if you gain 350 points for SC, it means 350 for Booty Bay, 350 for Everlook, 350 for Ratchet and 350 for Gadzetzan. Perfect display of such can be found on [Hellground DB](http://db.hellground.net/?quest=5525) there numbers are split correctly.

These mechanics of "united fractions" are widely represented in cloth-turning repeatable quests in every major city. Upon turning the quest player gain 75 reputation points for city fraction, and smaller amount (I believe its 15 or something) for other four fractions of Alliance/Horde.

This might also be a much wider issue, addressing several quests which solely give reputation with "Alliance" or "Horde". Will check those later.
